.docker/php-7.3/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM php:7.3-cli
+
+RUN apt-get update \
+    && apt-get install --assume-yes \
+        apt-utils \
+        git \
+        libzip-dev \
+        zip \
+        zlib1g-dev \
+    && apt-get clean --assume-yes \
+    && docker-php-ext-install \
+        zip
+
+RUN echo 'memory_limit = 256M' >> /usr/local/etc/php/conf.d/docker-php-memory-limit.ini
```

Comment:
To be safe, we should use `memory_limit=-1`, I think.

Reply:
Can do!